packages/database/src/migrations/20210913030808-AddCovidFiji7DayPositivityRateReport-modifies-data.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+import {
+  insertObject,
+  generateId,
+  findSingleRecord,
+  findSingleRecordBySql,
+  deleteObject,
+} from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const generateConfig = () => ({
+  code: `FJ_Covid_7_Day_Positivity_Rate`,
+  reportConfig: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'requested',
+          },
+        },
+      ],
+      dataElements: ['COVID_FJ_7_Day_Rolling_Pos_Tests', 'COVID_FJ_7_Day_Rolling_Num_Tests'],
+    },
+
+    transform: [
+      'keyValueByDataElementName',
+      {
+        transform: 'mergeRows',
+        groupBy: 'period',
+        using: {
+          COVID_FJ_7_Day_Rolling_Num_Tests: 'sum',
+          COVID_FJ_7_Day_Rolling_Pos_Tests: 'sum',
+          '*': 'exclude',
+        },
+      },
+      {
+        transform: 'excludeRows',
+        where:
+          '=not (exists($COVID_FJ_7_Day_Rolling_Num_Tests) and $COVID_FJ_7_Day_Rolling_Num_Tests > 0)',
```

Comment:
```suggestion
          '=not exists($COVID_FJ_7_Day_Rolling_Num_Tests) or $COVID_FJ_7_Day_Rolling_Num_Tests <= 0',
```

Reply:
(this is optional btw!)

Reply:
I'll make the change, but keep the brackets 😂. I don't trust it if there's no brackets (I know I should, but they just make me feel safe)